mlir/test/mlir-miopen-driver/conv2d_host_validation.mlir

```diff
@@ -0,0 +1,4 @@
+// RUN: mlir-miopen-driver -fil_layout=yxck -in_layout=nhwc -out_layout=nhwk -pv -c | mlir-rocm-runner --shared-libs=%rocm_wrapper_library_dir/librocm-runtime-wrappers%shlibext,%linalg_test_lib_dir/libmlir_runner_utils%shlibext --entry-point-result=void | FileCheck %s  
```

Comment:
Not yet.

Reply:
I cannot specify dimensions. The current mlir-miopen-driver.cpp always uses default dimensions with or without -p, as -p's default value is true.

Reply:
For argument, use -p=false and manually specify all arguments needed.

For other directions, I think it can be done by linalg.conv(). Do you have much vision into what next step is going to be? If there's a likelihood that you will continue to use the linalg.conv(), you should figure this out earlier than later.

Reply:
I tested -pv with different dimensions. Some passed. Some failed, and others crashed.
Passed:
 -in_channels=8  -batchsize=128 -out_channels=128 -in_h=16 -in_w=16 -fil_h=5 -fil_w=5
 -in_channels=8  -batchsize=64 -out_channels=128 -in_h=32 -in_w=32 -fil_h=3 -fil_w=3

Failed:
-in_channels=8  -batchsize=8 -out_channels=128 -in_h=32 -in_w=32 -fil_h=3 -fil_w=3
 -in_channels=8  -batchsize=16 -out_channels=64 -in_h=32 -in_w=32 -fil_h=3 -fil_w=3

Crashed:
 -in_channels=8  -batchsize=128 -out_channels=64 -in_h=32 -in_w=32 -fil_h=3 -fil_w=3
 -in_channels=8  -batchsize=128 -out_channels=64 -in_h=32 -in_w=32 -fil_h=3 -fil_w=3

Reply:
I doubt linalg.conv() can do other kinds of convolution, as I don't see its interface takes any argument that specify directions. But I may be missing something here.

Reply:
@yiqian1 For crashed cases, please provide the generated code as minimal reproducing example and clarify with @whchung. Failed cases might not be real failures, since I don't have any context on the precision of numeric here. `==` can be too strict to stand.

@yiqian1 Under the hood, convolution for bwd, wrw are basically also doing convolution under the hood. I don't see any reason linalg.conv cannot be served as the backend.

Reply:
Could you add those (at least 5) configs that pass to this unit test? The more strict we are the better we can catch regressions.